Automattic/Gravatar-SDK-iOS
# Adding Default Image Option to `UIImageView.gravatar.setImage`

Adding Default Image Option to `UIImageView.gravatar.setImage` with email method.

Also added the option to the UIImageView Extension demo.

**Note:** We have enough options now on the demo app that are overflowing the screen on SE devices.
I'll add a task to re-structure the demo screens.

### To test:

- Run the demo app
- Select `UIImageView Extension`
- Don't enter any email
- Tap on `Fetch Avatar`
  - The Gravatar logo should appear as the image (Backend default)
- Tap on `Default Image` button.
- Choose any of the given options
- Tap on `Fetch Avatar`
  - Check that the corresponding Default image appears.

closes #72
